### DISCUSSION TITLE
Solved "An existing connection was forcedly closed by the remote host OS ERROR 10054"

### DISCUSSION THREAD

**Discussion by nikkdevic** [rustdesk/rustdesk#3940] [Q&A] [answered]
I started getting this error after updating Windows.
"An existing connection was forcedly closed by the remote host OS ERROR 10054"
Sometimes I would also get "Remote desktop offline"

I downloaded the installer from website and updated(reinstalled) Rustdesk.
That fixed the problem.

I couldnt find this error or solution in any of the Issues so I am posting this for anyone else who might have this Issue.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/wiki/FAQ#force-relay
